app/views/components/link.scala.html

```diff
@@ -14,6 +14,14 @@
  * limitations under the License.
  *@
 
-@(href : String, id : String = "", key : String, openInNewWindow: Boolean = false)(implicit messages: Messages)
+@import viewmodels.Link
 
-<a id="@id" href="@href" @if(openInNewWindow) {target="_blank"}>@messages(key)</a>
\ No newline at end of file
+@(link: Link)(implicit messages: Messages)
+
+<li class="app-task-list__item" id="task-list__item--@{link.text}">
+    <span class="app-task-link">
+        <a class="app-task-list__task-name" id="task-list__task--@{link.text}" href="@{link.url}" aria-describedby="task-list__task--@{link.text}__tag">
```

Comment:
I don't believe the markup should have all these id's and css classes as we're not marking up a task list on this alternative choice page.